dev/com.ibm.ws.classloading.bells/src/com/ibm/ws/classloading/bells/internal/Bell.java

```diff
@@ -367,9 +361,10 @@ private static void getServiceInfos(ArtifactEntry providerConfigFile, String ser
 
             // Not entirely sure what these properties would be used for, but ...
             // they are currently used by the FAT tests to force all bell services to be eagerly created.
-            final Dictionary<String, Object> properties = new Hashtable<String, Object>();
+            final Hashtable<String, Object> properties = new Hashtable<String, Object>();
             properties.put("implementation.class", serviceInfo.implClass);
             properties.put("exported.from", library.id());
+            properties.putAll(serviceInfo.props);
```

Comment:
Move the "putAll" in front of the "put" calls so that there is no chance a customer can override the implementation.class and exported.from properties, b/c that would be a pain to diagnose.